chsdi/static/doc/examples/ol3_lv95_all.js

```diff
@@ -0,0 +1,173 @@
+var RESOLUTIONS = [
+    4000, 3750, 3500, 3250, 3000, 2750, 2500, 2250, 2000, 1750, 1500, 1250,
+    1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5, 0.25, 0.1
+];
+
+var extent = [2420000, 130000, 2900000, 1350000];
+
+
+var lang = getUrlParameter('lang') || 'en';
+var layer = getUrlParameter('layer') || 'ch.swisstopo.pixelkarte-farbe_wmts';
+var timestamp = getUrlParameter('timestamp') || 20140520;
+var format = getUrlParameter('format') || 'jpeg';
+var raw_url = getUrlParameter('base_url') || qualifyURL('..');
+
+var BASE_URL = raw_url.replace(/\/+$/, "")
+
+function getUrlParameter(sParam) {
+    var sPageURL = window.location.search.substring(1);
+    var sURLVariables = sPageURL.split('&');
+    for (var i = 0; i < sURLVariables.length; i++) {
+        var sParameterName = sURLVariables[i].split('=');
+        if (sParameterName[0] == sParam) {
+            return sParameterName[1];
+        }
+    }
+}
+
+function qualifyURL(url) {
+    var a = document.createElement('a');
+    a.href = url;
+    return a.cloneNode(false).href;
+}
+
+var matrixIds = [];
+for (var i = 0; i < RESOLUTIONS.length; i++) {
+    matrixIds.push(i);
+}
+
+
+
+var cadastralCfg = {
+    "attribution": "Mensuration officielle suisse / FL",
```

Comment:
minor: we don't need the double quotes here. `"attribution": "Mensuration officielle suisse / FL",` to `attribution: "Mensuration officielle suisse / FL",`